test/JDBC/upgrade/13_4/schedule

```diff
@@ -0,0 +1,52 @@
+# Schedule File for JDBC Test Framework for local run
+# 1. Lines starting with '#' will be treated as comments
+# 2. To run a postgres command:	cmd#!#postgresql#!#<enter postgres command>
+# 3. To run a T-SQL command: cmd#!#sqlserver#!#<enter T-SQL command>
+# 4. Keyword "all" is equivalent to running all test files in
+#    input folder
+# 5. To add a test, add test name (without extension Eg. BABEL-TEST) on a
+#    new line
+# 6. If you want the framework to not run certain files, use: ignore#!#<test name>
+
+TestBigInt
+TestReal
+TestBinary
+# TestRowVersion - not supported in 13_4
+TestBIT
+TestSmallDatetime
+TestChar
+TestSmallInt
+TestDatetime2
+TestSmallMoney
+TestDatetime
+TestSQLVariant
+TestDate
+TestText
+
+# Latest version of these files give protocol violation in 13_4 to 13_6 upgrade
+# since fix is in 13_7 so running their modified version for this upgrade path
+# TestDecimal
```

Comment:
let us remove these from schedule file, these are part of prepare/verify and self explanatory

Reply:
done